lib_nbgl/src/nbgl_use_case.c

```diff
@@ -322,7 +343,7 @@ static void displaySettingsPage(uint8_t page, bool forceFullRefresh)
 
     // override some fields
     content.title            = settingsTitle;
-    content.isTouchableTitle = touchableTitle;
+    content.isTouchableTitle = true;
```

Comment:
Why? Maybe this could be done at `nbgl_useCaseSettings` level enforcing the value when setting `touchableTitle` if not `TARGET_STAX`? But I'm not even sure this is wanted? The app should have the choice?

Reply:
To be confirmed but we strongly push to remove the concept of "simple title", and have the same behavior on Europa & Stax, with this "title" always used to go back or to quit settings